RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -34,12 +34,30 @@ namespace {
     return -1.f; 
   }
     
-  inline float chi2(reco::Track const & tk) { return tk.normalizedChi2();}
+  inline float chi2n(reco::Track const & tk) { 
```

Comment:
why not leave the old chi2 implementation and add chi2Modified1D or something?
``` (chi2+count1dhits)/float(ndof+count1dhits) ``` looks to me like a special variable, not necessarily related to chi2 distribution.